source/_integrations/telekomunikace_2n.markdown

```diff
@@ -0,0 +1,48 @@
+---
+title: 2N Telekomunikace
+description: Instructions on how to set up 2N Telekomunikace in Home Assistant.
+ha_category:
+  - Button
+  - Sensor
+  - Switch
+ha_release: 2023.1
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@elektr0nisch'
+ha_domain: telekomunikace_2n
+ha_platforms:
+  - button
+  - sensor
+  - switch
```

Comment:
Oh. I wasn't aware of that rule, but it makes sense, in order to decrease review work. I'll keep the button platform, because the functions controlled by these are supported by the most devices.